crates/driver/src/infra/config/file/mod.rs

```diff
@@ -148,8 +148,16 @@ fn default_soft_cancellations_flag() -> bool {
     false
 }
 
-fn default_http_time_buffer_milliseconds() -> u64 {
-    1500
+pub fn default_http_time_buffer_milliseconds() -> u64 {
+    500
+}
+
+pub fn default_solve_competition_time_buffer_percent() -> u64 {
+    20
+}
+
+pub fn default_quote_competition_time_buffer_percent() -> u64 {
+    20
```

Comment:
If we are not going to differentiate between the two, can we fold them into one value for #simplicity for now?

Reply:
Fixed.